plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/impl/MavenInvoker.java

```diff
@@ -79,7 +80,7 @@ private List<String> createGoalsList() throws IOException {
         return goals;
     }
 
-    private List<String> getActiveRecipes(List<String> recipes, JsonNode recipesNode) {
+    private List<String> getActiveRecipes(List<String> recipes, ArrayNode recipesNode) {
```

Comment:
Now that we have `RecipeDescriptor` why we don't use where it should instead of raw Jackson types ?

Also from what I understand from this method is that it return the list of coordinate of the active recipes ?

Shouldn't be called `getRecipesCoordinates` or something similar ?

Also using record is much more clear for a reader to understand

```java
private List<String> getRecipesCoordinates(List<RecipeDescriptor> availableRecipes,
                                           List<String> activeRecipes) {
```

For me I can be improved in a next PR

Reply:
Hey @jonesbusy,

> Also from what I understand from this method is that it return the list of coordinate of the active recipes ?

[getRecipesCoorinadates()](https://github.com/jenkinsci/plugin-modernizer-tool/blob/main/plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/impl/MavenInvoker.java#L117) returns the list of coordinate of active recipes, while [getActiveRecipes()](https://github.com/jenkinsci/plugin-modernizer-tool/blob/main/plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/impl/MavenInvoker.java#L104) returns the list of fqcn of recipes.

I have raised PR #71 to use record instead of raw Jackson types